plugins/modules/rds_cluster.py

```diff
@@ -189,12 +189,14 @@
         description:
           - Specifies the storage type to be associated with the DB cluster.
           - This setting is required to create a Multi-AZ DB cluster.
-          - When specified, a value for the I(iops) parameter is required.
+          - When specified I(storage_type=io1), a value for the I(iops) parameter is required.
```

Comment:
This is a suggestion
```suggestion
          - Multi-AZ DB clusters, I(storage_type) defaults to C(aurora) and a value for the I(iops) parameter is required.
```